src/lbaf/Model/lbsPhase.py

```diff
@@ -79,9 +79,19 @@ def get_number_of_objects(self):
 
         return self.__n_objects
 
+    def get_objects(self):
+
+        """ Return all objects belonging to phase."""
+
+        objects = []
+        for p in self.__ranks:
+            objects += p.get_objects()
```

Comment:
```suggestion
        objects = [p.get_objects() for p in self.__ranks]
```

Reply:
Actually no, because here `+` is used to perform a **list concatenation**; the suggested change results in creating a list of lists (which is not what we want).